src/lib/value-formatters.ts

```diff
@@ -25,7 +25,16 @@ export class TimeFormatter implements ValueFormatter {
   formatUnsigned(v: number) {
     const s = v * this.multiplier
 
-    if (s / 60 >= 1) return `${(s / 60).toFixed(2)}min`
+    if (s / 60 >= 1) {
```

Comment:
I think you want something like this:

```
if (s / 60 >= 1) {
  const minutes = Math.floor(s / 60)
  const seconds = Math.floor(s - minutes * 60)
  return `${minutes}:${zeroPad(seconds, 2)}`.
}
```

Where `zeroPad` is a function that you can import from `utils.ts`: https://github.com/jlfwong/speedscope/blob/828beb7ccf329ad3d3f6821ea53e577621ec6432/src/lib/utils.ts#L84

You can use `const` instead of `let` here because the variable is never reassigned. 

You might already know this, but in case you don't, you can read about this here: https://wesbos.com/let-vs-const/.